build.gradle.kts

```diff
@@ -30,6 +33,37 @@ allprojects {
         }
     }
 
+    apply(plugin = rootProject.libs.plugins.detekt.get().pluginId).also {
+        detekt {
+            buildUponDefaultConfig = true
+            allRules = true
+            parallel = true
+            config.from("${rootProject.projectDir}/detekt.yml")
+        }
+        tasks.withType<Detekt> {
+            setSource(files(project.projectDir))
+            exclude("**/build/**")
+            exclude {
+                it.file.relativeTo(projectDir).startsWith(
+                    project.layout.buildDirectory.asFile.get().relativeTo(projectDir)
+                )
```

Comment:
❓ What exactly should be excluded here?

Reply:
@oanamariabalaita it is the `build` directory depend of the gradle module, ksp generates `kotlin` files and generated code doesn't aligned with detekt standards, so we should exclude them.